otter/convergence/gathering.py

```diff
@@ -255,3 +272,20 @@ def get_all_launch_server_data(
         'lb_nodes': list(concat([clb, rcv3]))
     })
     return eff
+
+
+def get_all_launch_stack_data(
+        tenant_id,
+        group_id,
+        now,
+        get_scaling_group_stacks=get_scaling_group_stacks):
+    """
+    Gather all launch_stack data relevant for convergence w.r.t given time
+
+    Returns an Effect of ([HeatStack], None) to match
+    get_all_launch_server_data.
+    """
+    eff = (get_scaling_group_stacks(tenant_id, group_id, now)
+           .on(map(HeatStack.from_stack_details_json)).on(list)
+           .on(lambda stacks: {'stacks': stacks}))
```

Comment:
If this returning stacks in a structured way (`HeatStack` objects) then why make it a dict?